<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Kop" failures="1" tests="5" name="Default test" time="0.052" errors="0" timestamp="2021-10-15T23:58:22 ICT">
  <testcase classname="TH_15_10_2021_TODAY.AbstractClass" name="@BeforeTest testcaseSetUp" time="-1.634317102344E9">
    <failure type="org.testng.TestNGException" message="
Can inject only one of &amp;lt;ITestContext, XmlTest&amp;gt; into a @BeforeTest annotated testcaseSetUp.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Can inject only one of <ITestContext, XmlTest> into a @BeforeTest annotated testcaseSetUp.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:439)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:353)
at org.testng.internal.Parameters.createParameters(Parameters.java:708)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:168)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest testcaseSetUp -->
  <testcase classname="TH_15_10_2021_TODAY.AbstractClass" name="@AfterTest testCaseTearDown" time="-1.634317102392E9">
    <skipped/>
  </testcase> <!-- @AfterTest testCaseTearDown -->
  <testcase classname="TH_15_10_2021_TODAY.TCs_HandleTableFunctions" name="getTotalColumnNumberTest" time="0.0">
    <skipped/>
  </testcase> <!-- getTotalColumnNumberTest -->
  <testcase classname="TH_15_10_2021_TODAY.TCs_HandleTableFunctions" name="getRowNumberByValueTest" time="0.0">
    <failure type="org.testng.TestNGException" message="
Method public void TH_15_10_2021_TODAY.TCs_HandleTableFunctions.getRowNumberByValueTest() throws java.lang.InterruptedException requires a @DataProvider named : none">
      <![CDATA[org.testng.TestNGException: 
Method public void TH_15_10_2021_TODAY.TCs_HandleTableFunctions.getRowNumberByValueTest() throws java.lang.InterruptedException requires a @DataProvider named : none
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- getRowNumberByValueTest -->
  <testcase classname="TH_15_10_2021_TODAY.TCs_HandleTableFunctions" name="getTotalRowNumberTest" time="0.0">
    <skipped/>
  </testcase> <!-- getTotalRowNumberTest -->
  <testcase classname="TH_15_10_2021_TODAY.TCs_HandleTableFunctions" name="getTableCellValueTest" time="0.0">
    <skipped/>
  </testcase> <!-- getTableCellValueTest -->
  <testcase classname="TH_15_10_2021_TODAY.TCs_HandleTableFunctions" name="checkIfItemIsAppeared" time="0.0">
    <skipped/>
  </testcase> <!-- checkIfItemIsAppeared -->
</testsuite> <!-- Default test -->
